src/common/components/cards/SetSummary.tsx

```diff
@@ -74,6 +76,7 @@ class SetSummary extends React.Component<SetSummaryProps & WithStyles, SetSummar
           <strong>{name}</strong> by {metadata.authorName}
         </div>
         <div className={classes.controls}>
+          {this.renderButton('Create Deck', onCreateDeckFromSet, { disabled: cards.length < 15 })}
```

Comment:
Should `renderButton` be replaced with a component?

Reply:
I think so far it's simple enough that it doesn't need to be its own component.